src/Rings/mpoly-ideals.jl

```diff
@@ -273,9 +273,19 @@ ideal(z, x*y)
 function saturation(I::MPolyIdeal{T}, J::MPolyIdeal{T}) where T
   singular_assure(I)
   singular_assure(J)
-  K, _ = Singular.saturation(I.gens.S, J.gens.S)
+  K, _ = Singular.saturation2(I.gens.S, J.gens.S)
```

Comment:
```suggestion
  K, _ = Singular.saturation(I.gens.S, J.gens.S)
```
Due to this change `saturation` and `_saturation2` are exactly the same. I think this change here was not meant to be made.

Reply:
Oops, thanks!